boards/itsybitsy_m4/examples/sercom_interrupt.rs

```diff
@@ -0,0 +1,159 @@
+#![no_std]
+#![no_main]
+
+//! The idea of this example is to show two different things
+//! - How to manually configure SERCOM for a desired mode (UART in this case)
+//! - How to enable an use SERCOM interrupts
+//! The second one is particularly handy because some drivers will give
+//! you a `poll()` or `update()` function, and you are expected to call it  
+//! on the receive interrupt of the peripheral i.e. SERCOM UART RXC
+//! (receive complete flag).
+//!
+//! This example is basically just a blinky script. The MCU sends itself
+//! a byte via UART that will trigger the RXC interrupt and then toggles
+//! the builtin red led
+//!
+//! Wiring
+//! - RX - A4
+//! - TX - A5
+//!
+//! > In the ATSAMX5x MCUs SERCOM UART configuration requires at least
+//! two pins from the same IOSET. You can find more info about this in
+//! the hal documentation or in the MCU datasheet at:
+//! <https://www.microchip.com/en-us/product/ATSAMD51G19A>
+
+use itsybitsy_m4 as bsp;
+#[cfg(not(feature = "use_semihosting"))]
+use panic_halt as _;
+#[cfg(feature = "use_semihosting")]
+use panic_semihosting as _;
+
+use bsp::entry;
+use cortex_m::{peripheral::NVIC, prelude::_embedded_hal_blocking_delay_DelayMs};
```

Comment:
I'm not sure you're supposed to rely on prelude names... probably should just do `prelude::*`

Reply:
all of these example comments apply to both examples

Reply:
yeah this was I mistake, srry